src/components/SearchGridFilter.vue

```diff
@@ -1,6 +1,17 @@
 <template>
-  <div :class="{ 'search-filters': true,
+<div>
+  <div class="is-hidden-desktop">
+    <transition name="modal">
+        <div class="overlay">
+          <div class="modal">
```

Comment:
Can you fix these wrong indentation of elements? It's hard to understand the hierarchy of elements and makes it harder to fix the conflicts later

Reply:
Done .